spec/comment_spec.rb

```diff
@@ -2,19 +2,25 @@
 
 RSpec.describe Comment, type: :model do
   before(:example) do
-    @user = User.new(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
-    @post = Post.new(author: @user, title: 'Hello', text: 'This is my first post')
+    @user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @post = Post.create(author: @user, title: 'Hello', text: 'This is my first post')
   end
 
   subject { Comment.new(user: @user, post: @post) }
 
-  it 'name should be present' do
+  it 'user should be present' do
     subject.user = nil
     expect(subject).to_not be_valid
   end
 
-  it 'title should be present' do
+  it 'post should be present' do
     subject.post = nil
     expect(subject).to_not be_valid
   end
+
+  it 'increments comment_counter of associated post after save' do
+    expect do
+      subject.save!
+    end.to change { @post.reload.comment_counter }.by(1)
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.) and as per last reviewer mentioned, Please test the `update_comment_counter` method, not the comment counter itself as you did